app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/sent/SentListScreen.kt

```diff
@@ -21,22 +21,23 @@ import androidx.compose.foundation.layout.padding
 import androidx.compose.material3.Scaffold
 import androidx.compose.material3.Surface
 import androidx.compose.material3.Text
-import androidx.compose.material3.adaptive.navigationsuite.NavigationSuiteType
+import androidx.compose.material3.adaptive.currentWindowAdaptiveInfo
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import com.infomaniak.swisstransfer.ui.components.NewTransferFab
 import com.infomaniak.swisstransfer.ui.components.NewTransferFabType
 import com.infomaniak.swisstransfer.ui.theme.SwissTransferTheme
 import com.infomaniak.swisstransfer.ui.utils.PreviewMobile
 import com.infomaniak.swisstransfer.ui.utils.PreviewTablet
+import com.infomaniak.swisstransfer.ui.utils.isWindowSmall
 
 @Composable
-fun SentListScreen(
-    navType: NavigationSuiteType,
-) {
+fun SentListScreen() {
```

Comment:
The app changed a lot in another currently open PR _(#57)_, so it will possibly easier to rebase this one upon the other one.
I'll try to do that.

In the meantime, I'm blocking this PR.

Reply:
It's been rebased, all is good.